middleware/src/main/java/com/anytypeio/anytype/middleware/mappers/ToCoreModelMappers.kt

```diff
@@ -503,7 +504,8 @@ fun MDVView.toCoreModels(): DVViewer = DVViewer(
     hideIcon = hideIcon,
     coverFit = coverFit,
     coverRelationKey = coverRelationKey.ifEmpty { null },
-    defaultTemplateId = defaultTemplateId.ifEmpty { null }
+    defaultTemplate = defaultTemplateId.ifEmpty { null },
+    defaultObjectType = defaultObjectTypeId.ifEmpty { ObjectTypeIds.PAGE }
```

Comment:
in the future, we could use something like `DefaultDataViewObjectTypeProvider` to avoid using hard-coded object type ids

Reply:
I agree and we need to do the refactoring after Spaces